docs/workforce-management-framework/topics/pro-Using-Keycloak-implementation.adoc

```diff
@@ -0,0 +1,89 @@
+[id='enable-keycloak-integration-{chapter}']
+= Enable {WFM-RC-KeycloakName} Integration
+
+The {WFM-RC-NameLong} Demo Applications allow for Authentication and Access Management using
+link:http://www.keycloak.org/index.html[{WFM-RC-KeycloakName}].
+The {WFM-RC-NameLong} {WFM-RC-KeycloakName} integration is situated inside of the Demo Applications.
+
+By default, the {WFM-RC-KeycloakName} Authentication and Access Management is not setup with the Demo Applications.
+A {WFM-RC-NameLong} {WFM-RC-KeycloakName} Docker image has been created to allow easy deployment of a {WFM-RC-KeycloakName} server configured
+with the {WFM-RC-NameLong} demo data.
+
+== Run {WFM-RC-KeycloakName} Docker image
+
+The image is published to docker hub
+
+    docker run -p 8080:8080 feedhenry/raincatcher-keycloak
+
+For more information on this docker image see the docker image link:{WFM-RC-KeycloakDockerURL}{WFM-RC-Branch}/README.md[README.md].
+
+NOTE: If you wish to configure your own instance of {WFM-RC-KeycloakName} see the following
+link:{WFM-RC-KeycloakURL}getting_started/topics/first-boot.html[getting started guide]. You will need to import our Realm
+settings by importing link:{WFM-RC-KeycloakDockerURL}{WFM-RC-Branch}/data_files/raincatcher-realm.json[/data_files/raincatcher-realm.json].
+
+[[enabling-the-keycloak-demo-server]]
+== Enabling {WFM-RC-KeycloakName} on the Demo Server
+
+The following steps are for using the pre-configured docker image but will work for any instance of {WFM-RC-KeycloakName}
+provided you implement our Realm settings.
+
+- login into the {WFM-RC-KeycloakName} server via a browser using the following URL
+
+    http://localhost:8080/auth/
+
+- Click on `client` on the left menu
```

Comment:
Maybe we can remove these steps and just used the config that we defined below? I get that the realm config might change in the docker image, but since we have defined it here anyway, it might save the end user a few steps to get the config they need.

Reply:
removed